### PR TITLE
Dcm 197 delete apprentice

### PR DESCRIPTION
**Pull request by leewadhams**

### CODE REVIEW

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/SFA.DAS.ProviderApprenticeshipsService.Web.csproj [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <MvcBuildViews>false</MvcBuildViews>
     <UseIISExpress>true</UseIISExpress>
-    <IISExpressSSLPort>44309</IISExpressSSLPort>
+    <IISExpressSSLPort>44388</IISExpressSSLPort>
```
Comment (mgwilliam) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/SFA.DAS.ProviderApprenticeshipsService.Web.csproj]: aaaaggghhhh!

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Commitment/DeleteConfirmation.cshtml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    </label>
+                    <label class="block-label selection-button-radio" for="@Html.IdFor(m => m.DeleteConfirmed)-False">
+                        @Html.RadioButtonFor(m => m.DeleteConfirmed, false, new { @id = Html.IdFor(m => m.DeleteConfirmed) + "-False" })
+                        No, don't delete
+                    </label>
+                </fieldset>     
+            </div>
+            <button type="submit" class="button">Continue</button>
+            @Html.HiddenFor(m => m.ProviderId)
+            @Html.HiddenFor(m => m.HashedCommitmentId)
```
Comment (mgwilliam) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Commitment/DeleteConfirmation.cshtml]: shouldn't we be able to get this from the route?

File: src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -128,7 +128,7 @@
         <div id="content">
             @if (TempData.ContainsKey("InfoMessage"))
             {
-                <div>
+                <div class="panel panel-border-wide alert-default record-deleted-panel">
```
Comment (mgwilliam) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Shared/_Layout.cshtml]: "record-deleted-panel" is too specific for a general "info" message

Comment (rajeshdigital) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Shared/_Layout.cshtml]: will change it to deletion-alert

**Pull request merged**